src/app/pages/login/login.page.ts

```diff
@@ -27,15 +30,24 @@ export class LoginPage implements OnInit {
     });
   }
 
-  goToHomePage() {
+  signIn() {
     this.isSubmitted = true;
     if (!this.signInForm.valid) {
       console.log('Please provide all the required values!')
       return false;
     } else {
       console.log(this.signInForm.value)
     };
-    this.router.navigateByUrl('/home');
+    this.authService.signIn(this.signInForm.value['username'], this.signInForm.value['password'], (error, data) => {
```

Comment:
Another major change so that navigation and error messages are determined by AngularFire2/Firebase login success and failure.